src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+| **3. Gather feedback and iterate design**  | [**In Progress**](#feedback) |
+| **4. Origin trial**                        | [**In Progress**][ot]        |
+| 5. Launch                                  | Not Started                  |
+
+</div>
+
+## Using Web NFC {: #use }
+
+### Enabling support during the origin trial phase
+
+Web NFC will be available on Android as an origin trial in Chrome 81. The origin
+trial is expected to end in Chrome 84.
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Enabling via chrome://flags
+
+To experiment with Web NFC locally on Android, without an origin trial token,
+enable the `#enable-webnfc` flag in `chrome://flags`.
+
+<figure class="w-figure">
+  <img src="./chrome-flag.png" alt="Screenshot of the chrome://flags internal page to enable for Web NFC on Android">
+  <figcaption class="w-figcaption">Experimental flag for Web NFC on Android</figcaption>
+</figure>
+
+### Feature detection {: #feature-detection }
+
+Detecting if Web NFC is supported can be done by checking `NDEFReader` and
+`NDEFWriter` objects. Note that this does not guarantee that NFC hardware is
+available.
+
+```js
+if ('NDEFReader' in window) { /* Scan NFC tags */ }
+if ('NDEFWriter' in window) { /* Write NFC tags */ }
+```
+
+### Terminology {: #terminology }
+
+NFC stands for Near Field Communications, a short-range wireless technology
+operating at 13.56 MHz which enables communication between devices at a distance
+less than 10 cm and a transmission rate of up to 424 kbit/s.
+
+An NFC tag is a passive NFC device, powered by magnetic induction when an active
+NFC device (.e.g a phone) is in proximity range.
+
+<figure class="w-figure">
+  <img src="./nfc-tag.jpg" alt="Photo of a transparent NFC tag">
+  <figcaption class="w-figcaption">A transparent NFC tag</figcaption>
+</figure>
+
+The `NDEFReader` and `NDEFWriter` objects are the two entry points in Web NFC
+that expose functionality for preparing reading and/or writing actions that are
+fulfilled when an NDEF tag comes in proximity range. The `NDEF` in `NDEFReader`
+and `NDEFWriter` stands for NFC Data Exchange Format, a lightweight binary
+message format standardized by the [NFC Forum].
+
+The `NDEFReader` object is for acting on incoming NDEF messages from NFC tags
+while the `NDEFWriter` object is for writing NDEF messages to NFC tags within
+range.
+
+An NFC tag that supports NDEF is like a post-it note. Anyone can read it, and
+unless it is read-only, anyone can write to it. It contains a single NDEF
+message which encapsulates one or more NDEF records. Each NDEF record is a
+binary structure that contains a data payload, and associated type information.
+Web NFC supports the following NFC Forum standardized record types: empty, text,
+URL, smart poster, MIME type, absolute URL, external type, unknown, and local
```

Comment:
ou mix smart poster and smart-poster a bit

Reply:
Use 'smart poster'. It looks like [no one uses 'smart-poster'](https://trends.google.com/trends/explore?q=smart%20poster,smart-poster).

Reply:
Fixed.